pydarn/utils/conversions.py

```diff
@@ -114,11 +114,9 @@ def gate2slant(record, nrang, center=True):
     # Range offset
     # If center is true, calculate at the center
     if center:
-        # 0.5 off set to the centre of the range gate instead of edge
-        range_offset = -0.5 * record['rsep']
-    else:
         range_offset = 0.0
-
+    else:
+        range_offset = -0.5 * record['rsep']
```

Comment:
```suggestion
        range_offset = 0.5 * record['rsep']
```
Thinking about this, and definitely get a second opinion on this because it's friday afternoon and I could just be suffering from a case of fridays, if it's not at the center of the range gate, then do we want to add the range offset? So are we adding the range offset from the edge of the range gate thats closest to the radar (inner) or are we removing it from the outer edge of the range gate? @billetd @Devin-Huyghebaert

Reply:
Okay, so I think I'm just not understanding what the the center option is for. So it's default true, which means that the data is plotted at the center of the range-gate, if the user gives center=False then it switches to plotting at the start of the range gate, for which a - makes sense.

Reply:
```suggestion
        range_offset = 0.0
```